Sources/PackageGraph/RepositoryPackageContainer.swift

```diff
@@ -158,7 +162,9 @@ public class RepositoryPackageContainer: BasePackageContainer, CustomStringConve
     public override func getDependencies(at version: Version, productFilter: ProductFilter) throws -> [RepositoryPackageConstraint] {
         do {
             return try cachedDependencies(forIdentifier: version.description, productFilter: productFilter) {
-                let tag = knownVersions[version]!
+                guard let tag = try self.knownVersions()[version] else {
```

Comment:
This is great.  I don't know that there are any reported crashes due to this, but still...